src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesPublisher.java

```diff
@@ -198,7 +201,15 @@ private History createAnalysisHistory(final ResultSelector selector, final Repor
         if (referenceJobName != null) {
             Optional<Job<?, ?>> referenceJob = new JenkinsFacade().getJob(referenceJobName);
             if (referenceJob.isPresent()) {
-                baseline = referenceJob.get().getLastBuild();
+                if (referenceBuildId == null) {
+                    baseline = referenceJob.get().getLastBuild();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['}' at column 17 should be alone on a line.](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=4335332)